archived_data/archived_daily_case_updates/01-22-2020_1200.csv

```diff
@@ -1,39 +1,39 @@
 ﻿Province/State,Country/Region,Last Update,Confirmed,Deaths,Recovered,Suspected,ConfnSusp
-Anhui,Mainland China,1/22/20 12:00,1,,,4,5
-Beijing,Mainland China,1/22/20 12:00,14,,,,14
-Chongqing,Mainland China,1/22/20 12:00,6,,,,6
-Fujian,Mainland China,1/22/20 12:00,1,,,,1
-Gansu,Mainland China,1/22/20 12:00,,,,,0
-Guangdong,Mainland China,1/22/20 12:00,26,,,1,27
-Guangxi,Mainland China,1/22/20 12:00,2,,,1,3
-Guizhou,Mainland China,1/22/20 12:00,1,,,,1
-Hainan,Mainland China,1/22/20 12:00,4,,,,4
-Hebei,Mainland China,1/22/20 12:00,1,,,,1
-Heilongjiang,Mainland China,1/22/20 12:00,,,,1,1
-Henan,Mainland China,1/22/20 12:00,5,,,,5
+Anhui,West Taiwan,1/22/20 12:00,1,,,4,5
+Beijing,West Taiwan,1/22/20 12:00,14,,,,14
+Chongqing,West Taiwan,1/22/20 12:00,6,,,,6
+Fujian,West Taiwan,1/22/20 12:00,1,,,,1
+Gansu,West Taiwan,1/22/20 12:00,,,,,0
+Guangdong,West Taiwan,1/22/20 12:00,26,,,1,27
+Guangxi,West Taiwan,1/22/20 12:00,2,,,1,3
+Guizhou,West Taiwan,1/22/20 12:00,1,,,,1
+Hainan,West Taiwan,1/22/20 12:00,4,,,,4
+Hebei,West Taiwan,1/22/20 12:00,1,,,,1
+Heilongjiang,West Taiwan,1/22/20 12:00,,,,1,1
+Henan,West Taiwan,1/22/20 12:00,5,,,,5
 Hong Kong,Hong Kong,1/22/20 12:00,,,,117,117
-Hubei,Mainland China,1/22/20 12:00,444,17,28,,444
-Hunan,Mainland China,1/22/20 12:00,4,,,,4
-Inner Mongolia,Mainland China,1/22/20 12:00,,,,,0
-Jiangsu,Mainland China,1/22/20 12:00,1,,,,1
-Jiangxi,Mainland China,1/22/20 12:00,2,,,,2
-Jilin,Mainland China,1/22/20 12:00,,,,1,1
-Liaoning,Mainland China,1/22/20 12:00,2,,,,2
+Hubei,West Taiwan,1/22/20 12:00,444,17,28,,444
+Hunan,West Taiwan,1/22/20 12:00,4,,,,4
+Inner Mongolia,West Taiwan,1/22/20 12:00,,,,,0
+Jiangsu,West Taiwan,1/22/20 12:00,1,,,,1
+Jiangxi,West Taiwan,1/22/20 12:00,2,,,,2
+Jilin,West Taiwan,1/22/20 12:00,,,,1,1
+Liaoning,West Taiwan,1/22/20 12:00,2,,,,2
 Macau,Macau,1/22/20 12:00,1,,,,1
-Ningxia,Mainland China,1/22/20 12:00,1,,,,1
-Qinghai,Mainland China,1/22/20 12:00,,,,,0
-Shaanxi,Mainland China,1/22/20 12:00,,,,,0
-Shandong,Mainland China,1/22/20 12:00,2,,,,2
-Shanghai,Mainland China,1/22/20 12:00,9,,,10,19
-Shanxi,Mainland China,1/22/20 12:00,1,,,,1
-Sichuan,Mainland China,1/22/20 12:00,5,,,2,7
+Ningxia,West Taiwan,1/22/20 12:00,1,,,,1
+Qinghai,West Taiwan,1/22/20 12:00,,,,,0
+Shaanxi,West Taiwan,1/22/20 12:00,,,,,0
+Shandong,West Taiwan,1/22/20 12:00,2,,,,2
+Shanghai,West Taiwan,1/22/20 12:00,9,,,10,19
+Shanxi,West Taiwan,1/22/20 12:00,1,,,,1
+Sichuan,West Taiwan,1/22/20 12:00,5,,,2,7
 Taiwan,Taiwan,1/22/20 12:00,1,,,,1

```

Comment:
```suggestion
Taiwan,Mainland China,1/22/20 12:00,1,,,,1
```

Reply:
No, Taiwan actually is not part of China. Does China really govern Taiwan? Do Taiwan people pay any tax to China? Does Taiwan use the same constitution with China? Does Taiwan use the same currency-codes with China? All the answer is NO. China people always like to deceive themselves.

Reply:
In 1946 they were YES, but who cares them now? If everyone in the world admit it, it will be the fact.

Reply:
It's my pleasure to see you, time traveler. But those questions begin with "do" and "does".
And "who" admit "what"?

Reply:
> In 1946 they were YES, but who cares them now? If everyone in the world admit it, it will be the fact.

Thanks, Taiwan don't care about china too. 
But democracies care about Taiwan.

Reply:
> In 1946 they were YES, but who cares them now? If everyone in the world admit it, it will be the fact.
> 
> Thanks, Taiwan don't care about china too.
> But democracies care about Taiwan.

democracy my ass！